### PR TITLE
align hero image and description

### PR DESCRIPTION
**Pull request by Dinnall**
### What's this PR do?

This pull request...
Aligns hero image with the appropriate persona description  

### How should this be reviewed?

...

### Any background context you want to provide?

...

### Relevant tickets

References [Pivotal #](https://www.pivotaltracker.com/n/projects/2417735/stories/172823028).

### Checklist

- [ x ] This PR has been added to the relevant Pivotal card.
- [ x ] Added screenshots of front-end changes on small, medium, and large screens.
- [ ] Added appropriate feature/unit tests.
Screen Shot 2020-05-26 at 5 22 43 PM
Screen Shot 2020-05-26 at 5 31 59 PM
Screen Shot 2020-05-26 at 5 32 50 PM

### CODE REVIEW

**Comment by aaronschachter** [comment]
> Added screenshots of front-end changes on small, medium, and large screens.

Can you please add these into the PR description, so we don't need to manually test each size?

File: resources/assets/components/pages/QuizResultPage/QuizResultPage.js [hunk 1]
```diff
@@ -61,24 +62,30 @@ const QuizResultPage = ({ id }) => {
...
-              <h1 className="my-3 font-normal font-league-gothic color-white uppercase">
-                {linkBlockTitle}
-              </h1>
-              {assetId ? (
-                <ContentfulAsset id={assetId} width={400} height={300} />
-              ) : null}
-              <TextContent className="grid-full text-white">
-                {content}
-              </TextContent>
+        <article data-test="quiz-result-page">
```
Comment (aaronschachter) [inline resources/assets/components/pages/QuizResultPage/QuizResultPage.js:65]: StyleCI shows errors for Cypress tests failing. Changing this back to `data-testid` should likely do the trick.

File: resources/assets/components/pages/QuizResultPage/QuizResultPage.js [hunk 2]
```diff
@@ -61,24 +62,30 @@ const QuizResultPage = ({ id }) => {
...
+          <header role="banner" className="bg-blurple-500">
+            <div className="md:flex my-auto p-8">
+              <div className="md:w-1/4 flex-grow md:flex-col bg-bottom ">
+                {assetId ? (
+                  <ContentfulAsset id={assetId} width={400} height={300} />
```
Comment (aaronschachter) [inline resources/assets/components/pages/QuizResultPage/QuizResultPage.js:70]: We chatted about this in standups before -- I made up this height and width when I was prototyping.  Does this match what our design calls for?

Comment (Dinnall) [reply resources/assets/components/pages/QuizResultPage/QuizResultPage.js:70]: I checked in abstract but the contentful image and background are one. I ping Leah to find out the exact image size.

File: resources/assets/components/pages/QuizResultPage/QuizResultPage.js [hunk 3]
```diff
@@ -62,23 +63,27 @@ const QuizResultPage = ({ id }) => {
...
+                    {linkBlockTitle}
+                  </span>
+                </h1>
+                <div className="color-white">
+                  <TextContent>{content}</TextContent>
+                </div>
+              </div>
             </div>
           </header>
-          <div className="bg-white base-12-grid py-3 md:py-6">
```
Comment (aaronschachter) [inline resources/assets/components/pages/QuizResultPage/QuizResultPage.js:81]: 💣 Removing [`base-12-grid` here has made some unexpected changes]( https://dosomething.slack.com/archives/CUQMU4Q6B/p1590609383006900?thread_ts=1590607150.002700&cid=CUQMU4Q6B) that I didn't catch - can you please fix in a new PR?

**Pull request merged**